sdk/purview/Microsoft.Azure.Management.Purview/src/Microsoft.Azure.Management.Purview.csproj

```diff
@@ -5,11 +5,11 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Purview</PackageId>
     <Description>Azure Purview is a unified data governance service that helps you manage and govern your on-premises, multicloud, and software-as-a-service (SaaS) data. </Description>
-    <Version>1.0.0-beta.1</Version>
+    <Version>1.0.0.1</Version>
```

Comment:
The version number should 1.0.1.0. The assemblyinfo as well.

Reply:
updated